spring-integration-redis/src/test/java/org/springframework/integration/redis/RedisTest.java

```diff
@@ -33,36 +38,64 @@
 import org.springframework.integration.test.util.TestUtils;
 import org.springframework.messaging.Message;
 
+import io.lettuce.core.ClientOptions;
+import io.lettuce.core.SocketOptions;
+
 /**
+ * The base contract for all tests requiring a Redis connection.
+ * The Testcontainers 'reuse' option must be disabled,so, Ryuk container is started
```

Comment:
Typo in the Javadocs: space before `so` 😄 

I know, copy/paste from other place, but still let's fix at least here since we have a chance during review 😉